mobile/ios/ios-core.ts

```diff
@@ -867,7 +870,7 @@ class PosixSocket implements Mobile.IiOSDeviceSocket {
 							}
 						}
 					} catch (e) {
-						this.$logger.trace("Exeption thrown!!!");
+						this.$logger.trace("Exception thrown!!!");
```

Comment:
Consider showing the exception as well - this will aid in debugging and customer support.